src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_3001_4000.cpp

```diff
@@ -559,17 +559,25 @@ GUI_TEST_CLASS_DEFINITION(test_3112) {
 
     auto showOverviewButton = qobject_cast<QToolButton*>(GTAction::button(os, "Show overview"));
     CHECK_SET_ERR(showOverviewButton != nullptr, "Overview button is not found");
-    CHECK_SET_ERR(showOverviewButton->isChecked(), "Overview button is not checked");
+    // Overview is s not visible for too big size
+    // See MSAEditor::initActions()
+    CHECK_SET_ERR(!showOverviewButton->isChecked(), "Overview button is checked");
+
+    // Click the "Overview" button on the main toolbar
+    // Expected state: the task is canceled, the overview is shown.
+    GTWidget::click(os, showOverviewButton);
+    GTUtilsTaskTreeView::waitTaskFinished(os, 20000);
+    CHECK_SET_ERR(GTUtilsTaskTreeView::getTopLevelTasksCount(os) == 0, "0: There are unfinished tasks");
 
     // Modify the alignment.
     // Expected state: the overview task starts.
     GTUtilsMsaEditor::removeColumn(os, 5);
-    CHECK_SET_ERR(GTUtilsTaskTreeView::getTopLevelTasksCount(os) == 1, "1: There are no active tasks ");
+    CHECK_SET_ERR(GTUtilsTaskTreeView::getTopLevelTasksCount(os) == 1, "1: There are no unfinished tasks");
 
     // Click the "Overview" button on the main toolbar
     // Expected state: the task is canceled, the overview is hidden.
     GTWidget::click(os, showOverviewButton);
-    GTUtilsTaskTreeView::waitTaskFinished(os, 2000);
+    GTUtilsTaskTreeView::waitTaskFinished(os, 20000);
```

Comment:
This timeout should be as small as before: the overview computation algorithm was optimized in v42